stdlib.sh

```diff
@@ -804,6 +804,13 @@ layout_node() {
   PATH_add node_modules/.bin
 }
 
+# Usage: layout opam
+#
+# Sets environment variables from `opam env`.
+layout_opam() {
+  eval "$(opam env)"
```

Comment:
```suggestion
  eval "$(opam env "$@")"
```
Some users might want to pass extra flags.

Reply:
Indeed!